src/Microsoft.AspNet.Mvc.Core/ActionResults/ContentResult.cs

```diff
@@ -11,11 +13,17 @@ namespace Microsoft.AspNet.Mvc
 {
     public class ContentResult : ActionResult
     {
-        public string Content { get; set; }
+        private readonly MediaTypeHeaderValue DefaultContentType = new MediaTypeHeaderValue("text/plain")
+        {
+            Encoding = Encoding.UTF8
```

Comment:
We've defined a static class with these (`Encodings`) - you want to use UTF8 **without BOM**